app.js

```diff
@@ -80,7 +80,8 @@ app.post('/api/message', function (req, res) {
   // Send the input to the assistant service
   assistant.message(payload, function (err, data) {
     if (err) {
-      return res.status(err.code || 500).json(err);
+      var status = (err.code  !== undefined && err.code > 0)? err.code : 500;
```

Comment:
You can do 
```
res.status((err.code && err.code > 0) ? err.code : 500).json(err)
```

PLease don't use `var` use `const`

Reply:
fixed